UI/src/components/molecules/RipaLocation.vue

```diff
@@ -335,6 +390,42 @@ export default {
       ]
     },
 
+    latitudeRules() {
```

Comment:
Lat and Long rules don't seem to be working.  I'm getting the error message after clicking "current location"